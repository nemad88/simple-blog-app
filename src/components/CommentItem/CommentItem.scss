.comment-item {
  font-size: 2.4rem;
  background: #E7E7E7;
  -webkit-box-shadow: 9px 10px 20px -8px rgba(0,0,0,0.75);
  -moz-box-shadow: 9px 10px 20px -8px rgba(0,0,0,0.75);
  box-shadow: 9px 10px 20px -8px rgba(0,0,0,0.75);
  padding: .5rem;
  margin-bottom: 2rem;

  h3 {
    font-size: 1.8rem;
  }

  p{
    font-size: 1.6rem;
  }
}