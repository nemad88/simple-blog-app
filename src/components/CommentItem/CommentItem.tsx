import React from "react";
import {Comment} from "../../redux/comments";

interface Props {
    comment: Comment;
}

const CommentItem: React.FC<Props> = ({comment}) => {
    return (
        <div>
            <div>
                <h4>{comment.name}</h4>
                <p>{comment.body}</p>
                <div>{comment.postId}</div>
            </div>
        </div>
    )
}

export default CommentItem;