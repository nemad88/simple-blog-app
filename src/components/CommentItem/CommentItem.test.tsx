import React from "react";
import { shallow } from "enzyme";
import CommentItem from "./CommentItem";

it("expect to render CommentItem component", () => {
  expect(
    shallow(
      <CommentItem
        comment={{
          postId: 1,
          id: 1,
          name: "id labore ex et quam laborum",
          email: "Eliseo@gardner.biz",
          body:
            "laudantium enim quasi est quidem magnam voluptate ipsam eos\ntempora quo necessitatibus\ndolor quam autem quasi\nreiciendis et nam sapiente accusantium",
        }}
      />
    )
  ).toMatchSnapshot();
});
