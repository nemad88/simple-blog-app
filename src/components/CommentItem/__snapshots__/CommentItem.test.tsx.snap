// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CommentItem component 1`] = `
<div
  className="comment-item"
>
  <h3>
    id labore ex et quam laborum
  </h3>
  <p>
    laudantium enim quasi est quidem magnam voluptate ipsam eos
tempora quo necessitatibus
dolor quam autem quasi
reiciendis et nam sapiente accusantium
  </p>
</div>
`;
